test-framework/test_runner.go

```diff
@@ -67,11 +67,27 @@ type Result struct {
 //
 type ImportResolver func(location common.Location) (string, error)
 
+// FileResolver is used to resolve and get local files.
+// Returns the content of the file as a string.
+// Must be provided by the user of the TestRunner.
+//
+type FileResolver func(path string) (string, error)
+
 // TestRunner runs tests.
 //
 type TestRunner struct {
+
+	// importResolver is used to resolve imports of the *test script*.
```

Comment:
The test script here is the Test contract, right? Either way please clarify because I am unsure.

Reply:
the "test script" in this case is the script with the test cases. i.e: the .cdc file you would run as tests.